README.md

```diff
@@ -1,32 +1,127 @@
 
-Mudlet is a quality MUD client, designed to take mudding to a new level.
+## Key Features
 
-It’s a new breed of a client on the MUD scene – with an intuitive user interface, a specially designed scripting framework, and a very fast text display. That, along with cross-platform capability and an open-source development model result in a very likable MUD client.
+* Simple to use
+* Very, very fast
+* Active development
+* Big, thriving community
+* Powerful trigger engine
+* Rich Lua-based coding functionality (API)
+* Excellent script editor
+* 2D and 3D mapper with autowalk
+* Powerful and simple to use GMCP
+* Discord integration
+* Completely themable
+* Open-source and Free
+* Cross platform: Windows, macOS and Linux
+    * full compatibility: run same scripts on any of the above
 
-## Easy to use client
+<details>
+  <summary>Explain?</summary>
+  
+### Easy to use client
 
-We’re big on usability, and as such, creating an easy to use client and interface is one of the defining goals of the project. This applies to both the power users and plain gamers – everyone will feel at home with Mudlet, without having to waste too much time figuring out how to do something.
+We’re big on usability, and as such, creating an easy to use client and interface is one of the defining goals of the project. This applies to both the power users and usual gamers – everyone will feel at home with Mudlet, without having to waste too much time figuring out how to do something.
 
-## Designed for speed
+### Designed for speed
 
 Mudlet is designed and built to be very fast and efficient right from the start. Its scripting engine is designed to scale to large systems without bogging down – and the text display is designed to handle thousands of lines in under a second. All in all, we are very serious about Mudlet being quick – and take all measures to make it so.
 
-## Powerful Scripting
+### Powerful Scripting
 
 Mudlet features a scripting framework using [Lua](https://www.lua.org/) – a small, fast and efficient scripting language. This allows Mudlet to leverage the existing community and large ecosystem of existing packages for Lua without the many drawbacks of creating a Mudlet specific scripting language. Best of all, the Lua API is seamlessly integrated in Mudlet and shared by all aliases, triggers, scripts, keybindings, buttons and other Mudlet components.
 
-## Cross-Platform love
+### Cross-Platform love
+
+We believe in making Mudlet available to people on all major platforms, and we work on keeping cross-platform compatibility right from start. Mudlet is available on Linux (both 32bit and 64bit), Windows, and macOS; you may be able to run it on additional platforms as well.
+</details>
+
+## Download
+
+Download for free from [mudlet.org](https://www.mudlet.org/download/)
+
+## How To Use
+
+1. Download and double-click on Mudlet to run
+1. Pick an existing game and hit `Connect`
+1. ... or play one of your choosing:
```

Comment:
"... or play one of your choosing:" --> "... or play a game of your choosing:"

Reply:
Why repeat `game` twice?

Reply:
To improve understandability

Reply:
OK if someone gets confused by it, I'll fix it. I don't think it is confusing as it stands and I'd rather not have a text that sounds like a legal document :confused: